### PR TITLE
Add test cases for UnplanCommand and SelectCommand

### PR DESCRIPTION
**Pull request by seox123**

### CODE REVIEW

File: src/test/java/seedu/waddle/logic/commands/UnplanCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    }
+
+    @Test
+    public void execute_invalidMultiIndexDayList_throwsCommandException() {
+        UnplanCommand unplanCommand = new UnplanCommand(MULTI_INDEX_FIRST_UNSCHEDULED_ITEM);
+
+        assertCommandFailure(unplanCommand, model, UnplanCommand.MESSAGE_INVALID_INDEX_NUMBER);
+    }
+
+    @Test
```
Comment (CFSY) [inline src/test/java/seedu/waddle/logic/commands/UnplanCommandTest.java:56]: lgtm

File: src/test/java/seedu/waddle/logic/commands/UnplanCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+public class UnplanCommandTest {
+
+    private final Model model = new ModelManager(getTypicalWaddle(), new UserPrefs());
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new UnplanCommand(null));
+    }
+
+    @Test
```
Comment (CFSY) [inline src/test/java/seedu/waddle/logic/commands/UnplanCommandTest.java:32]: lgtm

File: src/test/java/seedu/waddle/logic/commands/UnplanCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+import seedu.waddle.model.ModelManager;
+import seedu.waddle.model.UserPrefs;
+import seedu.waddle.model.item.Item;
+import seedu.waddle.model.itinerary.Itinerary;
+
+public class UnplanCommandTest {
+
+    private final Model model = new ModelManager(getTypicalWaddle(), new UserPrefs());
+
+    @Test
```
Comment (CFSY) [inline src/test/java/seedu/waddle/logic/commands/UnplanCommandTest.java:27]: lgtm

**Review by CFSY — APPROVED** [review]
Everything looks good to merge

**Review by csheun — COMMENTED** [review]
LGTM

**Review by csheun — APPROVED** [review]
Good work Sean

**Pull request merged**